moduleroot/.github/PULL_REQUEST_TEMPLATE.md.erb

```diff
@@ -1,23 +1,22 @@
-<!--
-Thank you for your pull request. Please provide a description above and
-review the checklist below.
 
-Contributors guide: ./CONTRIBUTING.md
--->
+
 
 ## Checklist
-<!--
-Remove items that do not apply. For completed items, change [ ] to [x].
--->
 
-- [ ] Keep pull requests small so they can be easily reviewed.
+- [ ] PR contains a single logical change (to build a better changelog)
```

Comment:
```suggestion
- [ ] PR contains a single logical change (to build a better changelog).
```

Reply:
Actually, I'd rather avoid having it as a checklist. Maybe just add this line to the notes in the comment below?

Reply:
That's fine with me too...

Reply:
Nobody is going to read the notes, at least the checkbox is still visible after the PR is created unless you've actively deleted it.

Reply:
I agree. However since we have very few external collaborators and mostly do the PRs by ourselves, it would be an unnecessary check since we already follow this discipline. So it would be nothing more than a checkbox that is going to be immediately ticked anyway.
I would just tick it every time just like I ticked the "keep it small" checkbox

Reply:
Sure, we (== Tarazed, I guess) follow the discipline of keeping PRs to single logical changes. However, I've had to remind other VSHNeers to split their PRs into multiple logical changes before. So having the checkbox to remind people might help.

In any case, I don't really care that much, just a thought.

Reply:
Re-added